src/main/java/uk/gov/justice/digital/hmpps/whereabouts/repository/VideoLinkAppointmentRepository.kt

```diff
@@ -7,6 +7,6 @@ interface VideoLinkAppointmentRepository : CrudRepository<VideoLinkAppointment,
   override fun findAll(): Set<VideoLinkAppointment>
 
   fun findVideoLinkAppointmentByAppointmentIdIn(appointmentIds: Set<Long>): Set<VideoLinkAppointment>
-
+  fun findAllByAppointmentId(appointmentId: Long): Set<VideoLinkAppointment>
```

Comment:
wouldn't you only ever have one of these for a given id?

Reply:
replaced by findOne